### PR TITLE
Altino to FLU

### PR DESCRIPTION
**Pull request by omichel**
This PR depends on #3658.

### CODE REVIEW

File: projects/robots/saeon/worlds/altino.wbt [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+  position -1.711396296916839 -1.2279055391770919 0.5934737912697113
   follow "vehicle"
 }
 TexturedBackgroundLight {
 }
 Solid {
-  translation 1.4071558 0.05 1.210173
-  rotation 0 1 0 5.4977871
+  translation 0.234703 1.45571 0.05
+  rotation 0 0 -1 -5.307179586466759e-06
```
Comment (BenjaminDeleze) [inline projects/robots/saeon/worlds/altino.wbt:14]:
```suggestion
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you, online minors things:

- The box and Altino are not oriented as before in altino.wbt.
- The update of the documentation is missing.

**Comment by omichel** [comment]
I reshuffled the world layout and Altino orientation so that the robot has a pretty nice trajectory.

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Pull request merged**